.github/workflows/benchmark.yml

```
@@ -0,0 +1,32 @@
+name: benchmarks
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+on:
+  pull_request:
+    branches:
+      - master
+      - main
+    paths-ignore:
+      - 'LICENSE.md'
+      - 'README.md'
+      - 'docs/**'
+      - 'format/**'
+      - 'test/**'
+jobs:
+    benchmarks:
+        runs-on: ubuntu-latest
+        strategy:
+          fail-fast: false
+        steps:
+          - uses: actions/checkout@v3
+          - uses: julia-actions/setup-julia@v1
+            with:
+              version: "1"
+          - uses: julia-actions/cache@v1
+          - uses: julia-actions/julia-buildpkg@v1
+          - run: |
+              git fetch origin +:refs/remotes/origin/HEAD
+              julia --project=benchmark/ -e 'using Pkg; Pkg.instantiate(); Pkg.develop(PackageSpec(path=pwd()))'
```

```suggestion
              julia --project=benchmark/ -e '
                  using Pkg
                  Pkg.develop(PackageSpec(path=pwd()))
                  Pkg.instantiate()'
```
If you `develop` before you `instantiate`, it doesn't need to first try to install a released version of Loess.